crates/viewer/re_ui/src/design_tokens.rs

```diff
@@ -418,23 +428,36 @@ impl DesignTokens {
 
     /// Used by the "add view or container" modal.
     pub fn thumbnail_background_color(&self) -> egui::Color32 {
-        get_global_color(&self.json, "{Global.Color.Grey.250}")
+        self.color(ColorToken::gray(S250))
     }
 }
 
 // ----------------------------------------------------------------------------
 
-fn get_aliased_color(json: &serde_json::Value, alias_path: &str) -> egui::Color32 {
-    parse_color(get_alias_str(json, alias_path))
-}
-
-fn get_global_color(json: &serde_json::Value, global_path: &str) -> egui::Color32 {
-    parse_color(global_path_value(json, global_path).as_str().unwrap())
-}
-
-fn get_alias_str<'json>(json: &'json serde_json::Value, alias_path: &str) -> &'json str {
-    let global_path = follow_path_or_panic(json, alias_path).as_str().unwrap();
-    global_path_value(json, global_path).as_str().unwrap()
+/// Build the [`ColorTable`] based on the content of `design_token.json`
+fn load_color_table(json: &serde_json::Value) -> ColorTable {
+    fn get_color_from_json(json: &serde_json::Value, global_path: &str) -> egui::Color32 {
+        parse_color(global_path_value(json, global_path).as_str().unwrap())
+    }
+
+    ColorTable {
+        color_table: Hue::all()
+            .iter()
+            .map(|hue| {
+                let shade_map = Shade::all()
+                    .iter()
+                    .map(|shade| {
+                        let color = get_color_from_json(
+                            json,
+                            &format!("{{Global.Color.{hue}.{}}}", shade.as_u16()),
```

Comment:
Just implement `Display` for `Shade` instead? 🤷 

```suggestion
                            &format!("{{Global.Color.{hue}.{shade}}}"),
```